examples/Microsoft.Spark.FSharp.Examples/Microsoft.Spark.FSharp.Examples.fsproj

```diff
@@ -1,11 +1,12 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>net461;netcoreapp2.1</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1</TargetFrameworks>
     <RootNamespace>Microsoft.Spark.Examples</RootNamespace>
     <AssemblyName>Microsoft.Spark.FSharp.Examples</AssemblyName>
+      <IsPackable>False</IsPackable>
```

Comment:
(nit) the whitespace here is off

Reply:
You can create a `Directory.Build.props` file in the root `examples` folder and put this `<IsPackable>` setting in it, so it doesn't need to be duplicated across all examples.

---
In reply to: [278753970](https://github.com/dotnet/spark/pull/54#discussion_r278753970) [](ancestors = 278753970)

Reply:
Cool, this was the solution I was looking for. Thanks @eerhardt !

Reply:
Done